Tests/Integration/src/ContainerConfiguratorTrait.php

```diff
@@ -19,6 +20,10 @@ protected function configureContainer(ContainerConfigurator $container): void
             $container->import($confDir . '/{services}' . self::CONFIG_EXTS);
             $container->import($confDir . '/{services}_' . $this->environment . self::CONFIG_EXTS);
             $container->import($confDir . '/routing.yaml');
+
+            if (class_exists(MessageBusInterface::class)) {
```

Comment:
Same thing here : `class_exists` used on an interface instead of `interface_exists`